modules/ossm-sidecar-validate-kiali.adoc

```diff
@@ -0,0 +1,31 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-validating-sidecar_{context}"]
+= Validating sidecar injection
+
+The Kiali console offers several ways to validate whether or not your applications, services, and workloads have a sidecar proxy.
+
+.Missing sidecar badge
+image::ossm-node-badge-missing-sidecar.svg[Missing Sidecar badge]
```

Comment:
Just to be sure, the image in the preview is the missing sidecar badge and not the system telling me the image is missing. 
![image](https://user-images.githubusercontent.com/83772007/162497642-e4ce0dee-e3d9-466a-a30b-6b3534acb349.png)

Reply:
Yes, the purple badge is the icon.  =)